test/manipulation.test.js

```diff
@@ -1096,6 +1096,157 @@ describe('manipulation', function() {
       });
     });
   }
+  if (!getSchema().connector.replaceById) {
+    describe.skip('replaceById - not implemented', function() {});
+  } else {
+    describe('replaceOrCreate when forceId is true', function() {
+      var Post;
+      var ds = getSchema();
+      before(function(done) {
+        Post = ds.define('Post', {
+          title: {type: String, length: 255, index: true},
+          content: {type: String},
+          comments: [String],
+        }, {forceId: true});
+        ds.automigrate('Post', done);
+      });
+
+      it('fails without options on create', function(done) {
```

Comment:
replaceOrCreate does not fail on all create calls - only when the data includes `id`. Please update the test name and also add a test for the case where the create operation succeeds.